From cdf5c86913e6138856032ffbe758ab44e5ff6126 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Fri, 7 Aug 2009 10:20:44 +0200
Subject: [PATCH 01/33] updated HP suspend quirks with s2ram whitelist entries from -r1.136

Updated HP suspend quirks with s2ram whitelist entries from -r1.136.
Added new:
- HP Compaq nx6315
---
 .../10freedesktop/20-video-quirk-pm-hp.fdi         |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi b/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
index 4a3016c..40fc620 100644
--- a/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
+++ b/fdi/information/10freedesktop/20-video-quirk-pm-hp.fdi
@@ -133,6 +133,12 @@
 	  </match>
         </match>
       </match>
+      <match key="system.hardware.product" contains="Compaq nx6315">
+        <match key="system.hardware.version" prefix="68MOU">
+    	  <merge key="power_management.quirk.s3_bios" type="bool">true</merge>
+	  <merge key="power_management.quirk.s3_mode" type="bool">true</merge>
+        </match>
+      </match>
       <match key="system.hardware.product" contains="nx 7010">
     	<merge key="power_management.quirk.s3_bios" type="bool">true</merge>
         <match key="system.hardware.product" contains="DU394T#">
-- 
1.7.1

